pkg/manager/manager.go

```diff
@@ -307,6 +313,7 @@ func New(config *rest.Config, options Options) (Manager, error) {
 		port:                  options.Port,
 		host:                  options.Host,
 		certDir:               options.CertDir,
+		webhookTLSConfig:      options.WebhookTLSConfig,
```

Comment:
What is your usecase for needing this and not being able to use the `certDir`?

Reply:
We have an internal library that we're required to call which produces a `tls.Config` that complies to a company standard.  Rather than going through a song-and-dance trying to convert that to files which would just be converted back to a `tls.Config` I'd prefer to just use the config directly.

Reply:
@alvaroaleman I want to setting custom TLS config, such as CipherSuites, so i think this feature is helpful for me. Or have other way can do it?